cmd/tx.go

```diff
@@ -898,7 +898,18 @@ $ %s tx raw send ibc-0 ibc-1 100000stake cosmos1skjwj5whet0lpe65qaq4rpq03hjxlwd9
 
 			// Query all channels for the configured connection on the src chain
 			srcChannelID := args[4]
-			channels, err := src.ChainProvider.QueryConnectionChannels(cmd.Context(), srch, path.Src.ConnectionID)
+			srcChainID := args[0]
```

Comment:
arg[0] is the chain name, not chain_id. This line can be removed

Reply:
I agree with you